src/AbstractActionHandler.test.ts

```diff
@@ -562,8 +566,9 @@ describe('Action Handler', () => {
     }
     const versionedActions = await actionHandler._applyUpdaters({}, nextBlock, {},  false)
     await actionHandler._runEffects(versionedActions, {}, nextBlock)
-    expect(startThrownEffect).toHaveBeenCalled()
-    expect(actionHandler.info.numberOfRunningEffects).toEqual(0)
+    expect(startThrownEffect).toBeCalledTimes(1)
+    expect(actionHandler.info.numberOfRunningEffects).toEqual(1)
+    expect(runEffect).toBeCalledTimes(2)
     expect(actionHandler.info.effectErrors).toHaveLength(1)
     expect(actionHandler.info.effectErrors![0].startsWith('Error: Thrown effect')).toBeTruthy()
```

Comment:
Why `effectErrors![0]` (optional)? `.toBe(true)` should be exact value you want here.